x-pack/test/functional/apps/saved_objects_management/spaces_integration.ts

```diff
@@ -32,15 +33,19 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
   describe('spaces integration', () => {
     before(async () => {
-      await esArchiver.load(
-        'x-pack/test/functional/es_archives/saved_objects_management/spaces_integration'
+      await spacesService.create({ id: spaceId, name: spaceId });
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management/spaces_integration',
+        { space: spaceId }
+      );
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/saved_objects_management/spaces_integration_space_1',
+        { space: spaceId }
       );
```

Comment:
I don't think it makes sense to have 2 kbn_archives here.  They both go into the same space.  One of these has the index pattern and the other has a pie chart which references that index pattern.

Reply:
Less is best, so I'll give 'er a go!